specification/ai/Azure.AI.Projects/tspconfig.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  "service-dir":
+    default: "sdk/ai"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "data-plane"
+    emit-lro-options: "none"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/azure-ai-projects.json"
+  "@azure-tools/typespec-python":
+    package-mode: "dataplane"
+    package-dir: "azure-ai-projects"
+    package-name: "{package-dir}"
+    flavor: azure
+    generate-test: false
+    generate-sample: false
+    examples-dir: "{project-root}/examples"
+  "@azure-tools/typespec-csharp":
+    package-mode: "dataplane"
+    package-dir: "Azure.AI.Projects"
+    namespace: "Azure.AI.Projects"
+    package-name: "{package-dir}"
+    model-namespace: false
+    flavor: azure
+    generate-test: false
+    generate-sample: false
+    examples-dir: "{project-root}/examples"
+  "@azure-tools/typespec-ts":
+    package-dir: "ai-projects"
+    generateTest: true
+    generateMetadata: false
```

Comment:
JS automation will clean up all the files before generation, so please enable ```generateMetadata```

Reply:
@dargilco you can disable it when you prepare the JS sdk release pr, but for pipeline, since you have customization code, we can't make sure they are compatible in the pipeline automation, which is why we have to clean up those to make sure the generated code works fine. Hence generateMetadata should be enabled in the pipeline.